app/models/event.rb

```diff
@@ -0,0 +1,47 @@
+class Event
+
+  attr_reader :name, :start_datetime, :end_datetime, :description, :location, :link, :thumbnail
+
+	def initialize(event_json)
+	  @name = event_json['summary']
+	  @start_datetime = event_json['start']
+	  @end_datetime = event_json['end']
+	  @location = event_json['location']
+	  @description = event_json['description']
+	  @link = (event_json['description'].to_s).split.first
```

Comment:
I have `#to_s` because, when I don't have it, it gives me a `NoMethodError: undefined method 'split' for nil:Nilclass` error.